_data/monthly-calls.yml

```diff
@@ -0,0 +1,88 @@
+    subtitle: Stepping back to look at the big picture
+    description: "It’s been eight years since the first public release of USWDS. What have we done? Where are we going? What have we learned? We revisited our mission and vision as we looked out in the direction of the next eight years."
+    date: September 2023
+    id: rl5hgoOK22o?si=mnOvqLh4ciC6JNzd
+    event_link: https://digital.gov/event/2023/09/21/uswds-monthly-call-september-2023/
+    slides:
+      link: https://s3.amazonaws.com/digitalgov/static/uswds-monthly-call-september-2023.pptx
+      size: 3.4
+      pages: 99
+  - title: Product roadmap
+    subtitle: Looking ahead to 2024 for the U.S. Web Design System
+    date: August 2023
+    id: Kj-MYq_0A3s?si=Ot2wE-Aw6IPXKqA_
+    event_link: https://digital.gov/event/2023/08/17/uswds-monthly-call-august-2023/
+    slides:
+      link: https://s3.amazonaws.com/digitalgov/static/uswds-monthly-call-august-2023.pptx
+      size: 3.9
+      pages: 94
+  - title: Introducing critical checklists
+    subtitle: Developing and evaluating content for manual accessibility and user experience (UX) tests
+    description: "We talked about the easy-to-follow, manual accessibility and user experience (UX) tests we’re developing for USWDS components. We’re calling these tests Critical Checklists."
+    date: June 2023
+    id: UxC-a48Vn7w?si=sJU-EAZmJfVRMEc-
+    event_link: https://digital.gov/event/2023/06/15/uswds-monthly-call-june-2023/
+    slides:
+      link: https://s3.amazonaws.com/digitalgov/static/uswds-monthly-call-june-2023.pptx
+      size: 2.0
+      pages: 79
+  - title: Real-world accessibility
+    subtitle: What's it like to be blind and use the web in 2023
+    description: "To observe Global Accessibility Awareness Day we talked about the real-world experience of blind users on the web."
+    date: May 2023
+    id: SG6Bbv-uKeg?si=9nzftzWV-V9MEtzQ
+    event_link: https://digital.gov/event/2023/05/18/uswds-monthly-call-may-2023/
+    slides:
+      link: https://s3.amazonaws.com/digitalgov/static/uswds-monthly-call-may-2023.pptx
+      size: 5.4
+      pages: 23
+  - title: Top tasks
+    subtitle: What the U.S. Web Design System team did and what they found
+    description: "We discussed how and why USWDS team performed their Top Tasks research, as well as what they learned and what they’ll do next."
+    date: April 2023
+    id: d1wPuISaHIg?si=Zx-IcO31nqL4m0C9
+    event_link: https://digital.gov/event/2023/04/20/uswds-monthly-call-april-2023/
+    slides:
+      link: https://s3.amazonaws.com/digitalgov/static/uswds-monthly-call-april-2023.pptx
+      size: 3.2
+      pages: 72
+  - title: "Making sense: Attention and the brain"
+    subtitle: Cognition, perception, and design
+    description: "The Design System team took a step back and explored how our design decisions both support and are supported by the ways our bodies and brains perceive order and make sense of the world."
```

Comment:
@amyleadem  Use lowercase for "design system" here ("The design system team took a step back..."). This isn't something we are doing consistently on the site right now, but we can at least be consistent on this page. Our existing style guides use lowercase.

Reply:
Updated in [43b65e9](https://github.com/uswds/uswds-site/pull/2345/commits/43b65e9e9d18fe84aa856ca94d589da9528efba7)